news/2017-10-09-halloween-fanart-contest.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Halloween 2017 Fanart Contest"
+date: 2017-10-09 06:00:00 +0000
+---
+
+Spooks, chills and thrills abound! Halloween is drawing close again, and you know what that means - a fanart contest! Two hectic weeks to pen any of the osu! mascots (or community mascots) in a delightfully spooky Halloween setting, for a chance at getting your work added to the game as a seasonal background!
```

Comment:
`close again and`
remove comma after `again` but before `and`??

Reply:
commas typically denote pauses - picture some gaudy announcer reading this out

halloween is drawing close again, (BREATH) and you know what that means (SMALLER PAUSE) a fanart contest!